docs/source/gbd2017_models/causes/ischemic_heart_disease/index.rst

```diff
@@ -147,22 +150,46 @@ State and Transition Data Tables
      - Measure
      - Value
      - Notes
-   * - C
-     - prevalence
-     - prevalence_c493
+   * - All
+     - cause-specific mortality rate
+     - :math:`\frac{\text{deaths_c493}}{\text{population}}`
+     -
+   * - P
+     - excess mortality rate
+     - emr_m15755
      -
-   * - C
+   * - A
      - excess mortality rate
-     - :math:`\frac{\text{deaths_c493}}{\text{population} \,\times\, \text{prevalence_c493}}`
+     - emr_m1814
      -
-   * - C
+   * - S
+     - excess mortality rate
+     - 0
+     -
+   * - P
      - disability weight
-     - :math:`\displaystyle{\sum_{s\in \text{sequelae_c493}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+     - :math:`\displaystyle{\sum_{s=380}^{385}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
      -
-   * - All
-     - cause-specific mortality rate
-     - :math:`\frac{\text{deaths_c493}}{\text{population}}`
+   * - A
+     - disability weight
+     - :math:`\scriptstyle{\text{disability_weight_378} \,\times\, \text{prevalence_378} + \text{disability_weight_379} \,\times\, \text{prevalence_379}}`
```

Comment:
This table is going off the page for me. I think if you write it like this it won't:

:math:`\displaystyle{\sum_{s\in \{378,379\}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`